src/player.h

```diff
@@ -43,6 +43,15 @@ namespace Player {
 		EngineEnglish = 8
 	};
 
+	/** Bitmask for activated patches */
+	enum PatchType {
+		PatchNone = 0,
+		/** DynRPG */
+		PatchDynRpg = 1,
+		/** ManiacPatch */
+		PatchManiacPatch = 2
```

Comment:
Why not just `PatchManiac`

Reply:
no particular reason, Patch is the enum prefix and ManiacPatch the patch name ;). But I changed it.